src/main/kotlin/dk/cachet/carp/webservices/study/controller/StudyController.kt

```diff
@@ -64,7 +64,6 @@ class StudyController
 
     private val studyService = coreStudyService.instance
 
-
```

Comment:
This is not a problem, but now that I see the lines that you are removing, maybe it would be wise for us to set up a project formatting rule. Core uses Detekt, maybe we could just steal that and use it here as well. I will create a trello card for this.

Reply:
Good idea, I mean I always try to format the code in a way that what's default. Never used Detekt, might look into it. Sure, trello card is needed.